app/src/androidTest/java/com/mapbox/maps/testapp/BaseMapTest.kt

```diff
@@ -38,21 +37,6 @@ abstract class BaseMapTest {
     loadMap()
   }
 
-  @After
-  fun tearDown() {
-    val latch = CountDownLatch(1)
-    rule.scenario.onActivity {
-      it.runOnUiThread {
-        mapView.onStop()
-        mapView.onDestroy()
-        latch.countDown()
-      }
-    }
-    if (!latch.await(10000, TimeUnit.MILLISECONDS)) {
-      throw TimeoutException()
-    }
-  }
-
```

Comment:
Could you explain the reason for this change, changes like these are great to highlight in your PR description.

Reply:
Yeah, with the lifecycle plugin, `mapView.onDestroy()`  will be invoked automatically, if we keep this part it will be invoked twice.